src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -464,171 +468,240 @@ subroutine init_atm_static(mesh, dims, configs)
        call mpas_log_write('*****************************************************************', messageType=MPAS_LOG_ERR)
        call mpas_log_write('Please correct the namelist.', messageType=MPAS_LOG_CRIT)
  end select surface_input_select1
- nx = 1200
- ny = 1200
- nz = 1
- isigned  = 1
- endian   = 0
- wordsize = 1
- scalefactor = 1.0
- allocate(rarray(nx,ny,nz))
- allocate(ncat(ismax_lu,nCells))
+
+ ierr = mgr % init(trim(config_geog_data_path)//trim(geog_sub_path))
+ if (ierr /= 0) then
+    call mpas_log_write('Error initalizing lu_index Geotile Manager', messageType=MPAS_LOG_CRIT)
+ endif
+
+ allocate(ncat(ismax_lu, nCells))
```

Comment:
Instead of using `ismax_lu` as the upper bound for `ncat`, should we use `category_min:category_max` so we have a lower and an upper bound for categories as specified in the index files?